### PR TITLE
Evinitsky/straight road pr

### PR DESCRIPTION
**Pull request by eugenevinitsky**
add a straight road example for faster training
    swap out the reward function with one that actual changes as a function of speed

### CODE REVIEW

File: flow/core/params.py [hunk 1]
```diff
@@ -637,6 +637,8 @@ class EnvParams:
         specifies whether to clip actions from the policy by their range when
         they are inputted to the reward function. Note that the actions are
         still clipped before they are provided to `apply_rl_actions`.
+    done_at_exit : bool, optional
+        If true, done is returned as True when the vehicle exits.
```
Comment (AboudyKreidieh) [inline flow/core/params.py:641]:
```suggestion
        If true, done is returned as True when the vehicle exits. This is only 
        applied to multi-agent environments.
```

Comment (eugenevinitsky) [reply flow/core/params.py:641]: Good catch.

**Comment by AboudyKreidieh** [comment]
Finished first round of reviews. Let me know when things are updated and I'll look at it again.

**Comment by eugenevinitsky** [comment]
Am I missing some comments? Seems like there's just the one comment.

**Comment by AboudyKreidieh** [comment]
woops, needed to submit them. they're up now

**Comment by eugenevinitsky** [comment]
Thanks for the review! Comments have been addressed.

**Pull request merged**